region/maridia/inner-pink/Colosseum.json

```diff
@@ -1194,6 +1194,283 @@
+                  "canMomentumConservingTurnaround",
+                  "canMomentumConservingMorph"
+                ]}
+              ],
+              "note": ["This applies to Flyway"],
+              "devNote": [
+                "This requires a last-frame jump, with a 3-frame window for the turnaround.",
+                "A ceiling mockball is also possible, with somewhat greater difficulty:",
+                "it needs a last-frame jump, with a 2-frame window for the morph."
+              ]
+            }
+          ]
+        }
+      },
+      "requires": [
+        "canCrossRoomJumpIntoWater",
+        "canTrickyJump"
+      ]
+    },
+    {
+      "link": [2, 3],
+      "name": "Side Platform Cross Room Jump into Spring Ball Jump",
+      "entranceCondition": {
+        "comeInWithSidePlatform": {
+          "platforms": [
+            {
+              "minHeight": 1,
+              "maxHeight": 1,
+              "minTiles": 4,
+              "speedBooster": false,
+              "obstructions": [[1, 0]],
+              "note": "Applies to Skree Boost Room, Screw Attack Room, and Lava Dive"
+            },
+            {
+              "minHeight": 2,
+              "maxHeight": 2,
+              "minTiles": 5.4375,
+              "speedBooster": false,
+              "obstructions": [[1, 0]],
+              "requires": [
+                "canTrickySpringBallJump",
+                "canTrickyJump"        
+              ],
+              "note": "Applies to Early Supers Room, Blue Hopper Room, Bowling Alley (bottom), Dust Torizo Room, and Noob Bridge"
+            },
+            {
+              "minHeight": 3,
+              "maxHeight": 3,
+              "minTiles": 9.4375,
+              "speedBooster": false,
+              "obstructions": [[1, 0]],
+              "requires": [
+                "canTrickySpringBallJump",
+                "canTrickyJump"
+              ],
+              "note": "Applies to Bomb Torizo Room, Pink Brinstar Hopper Room, Phantoon's Room, Big Boy Room, and Double Chamber."
+            },
+            {
+              "minHeight": 3,
+              "maxHeight": 3,
+              "minTiles": 9.4375,
+              "speedBooster": false,
+              "obstructions": [[3, 2]],
+              "requires": [
+                "canTrickySpringBallJump",
+                "canTrickyJump",
+                "canJumpIntoIBJ",
+                {"enemyDamage": {"enemy": "Mochtroid", "type": "contact", "hits": 1}}
+              ],
+              "note": "Applies to Metal Pirates Room."
+            },
+            {
+              "minHeight": 2,
+              "maxHeight": 2,
+              "minTiles": 31,
+              "speedBooster": true,
+              "obstructions": [[3, 0]],
+              "requires": [
+                "canInsaneJump",
+                {"or": [
+                  "canMomentumConservingTurnaround",
+                  {"and": [
+                    "canMomentumConservingMorph",
```

Comment:
I found the morph to be easier.

But a LeaveWith springball bounce should be the way you traverse these nodes.  Even better with a unmorph momentumConservingMorph.